test/spec/OktaUserAgent.ts

```diff
@@ -1,63 +1,118 @@
+const SDK_VERSION = (global as any).SDK_VERSION;
+const NODE_VERSION = (global as any).NODE_VERSION;
```

Comment:
Could we augment the `global` type locally with these variables so we don't need to cast to `any`?